src/detection.h

```diff
@@ -19,9 +19,8 @@
 #define SENSOR_FRONT_RIGHT_ID 3
 #define NUM_SENSOR 4
 
-struct Walls_info {
+struct sides_info {
```

Comment:
Maybe it is not needed anymore?

Can you create a PR with this commit only? :blush:

Reply:
#146